File: Day-27/Powerof2bruteforce.java
public class Powerof2bruteforce {
  
    public boolean isPowerOfTwo(int n) {
       StringBuilder sb=new StringBuilder();
       while(n>0)
       {
        int rem=n%2;
        sb.append(rem);
        n=n/2;
       }
       String str=sb.reverse().toString();
       int cnt=0;
       for(int i=0;i<str.length();i++)
       {
        if(str.charAt(i)=='1')
        {
            cnt++;
        }
        
       }
       return cnt==1;
    }
}

File: Day-27/Powerof2bruteforce.py
class Solution:
    def isPowerOfTwo(self, n: int) -> bool:
        cnt=0
        strs=[]
        while n>0:
            rem=n%2
            strs.append(str(rem))
            n=n//2
        for i in strs:
            if i=='1':
                cnt+=1
        return cnt==1
        